build/transactions/gas-calculation.mdx

```diff
@@ -0,0 +1,36 @@
+# Qi Transactions
+
+## Gas Calculation
+
+Gas calculation for Qi Transactions is unique and takes into account several factors specific to Quai Network's architecture. Here's an overview of how gas is calculated:
+
+### Total Gas Calculation
+
+The total gas for a Qi transaction is calculated by considering several components:
+
+1. **Intrinsic Gas**: This is a base cost associated with the transaction itself.
+2. **Input and Output Costs**: Each input and output in the transaction incurs a gas cost.
```

Comment:
I think #2 should be removed since it does not have a sub section of its own and is mentioned in the "Intrinsic Gas" sub section anyway.